### ISSUE TITLE
[Event modification request] CodeUnit 52 "BOM-BOM Component" 'OnAfterInsertEvent'

### ISSUE THREAD

**Issue by SweconThomasH** [microsoft/AL#6982]

**Comment by dnpb** [comment]
How is this an issue for the language or compiler?

**Comment by SweconThomasH** [comment]
@dnpb You are right, this is an issue in the app, in the **Base app**, to be exact. I would expect the Base app to show concise behaviour. 
Maybe you can explain the reasoning behind the fact that the OnAfterInsertEvent does **not** honour the parameter, the OnAfterDeleteEvent however, just 6 lines further down in the very same CodeUnit 52, **does** honour the parameter?

**Comment by dnpb** [comment]
I can't explain anything; I'm just another user.
